core/server/master/src/main/java/alluxio/master/file/meta/Inode.java

```diff
@@ -330,6 +332,21 @@ public T setMode(short mode) {
     return getThis();
   }
 
+  /**
+   * Sets ACL entries into the internal ACL.
+   * If an entry is new, it is added into the internal ACL;
```

Comment:
Is this correct? This will be a union and overwrite operation? Instead, should `set` be an actual set? (just use the input acl as the inode acl)

Reply:
reworded to `The entries will overwrite any existing correspondent entries in the internal ACL`.